src/config.template.yaml

```diff
@@ -132,8 +132,6 @@ platforms:
     min_seconds_to_record: 3
 
   dummy:
-    # whether to print messages when methods are called
-    verbose: false
```

Comment:
> permutation of contract

What do you mean by this?

Well, if you look at the code, `self._pconfig` is _always_ a dict.